unison-src/transcripts/destructuring-binds.output.md

```diff
@@ -86,21 +86,17 @@ ex4 =
 
   Loading changes detected in scratch.u.
 
-  I couldn't find any definitions matching the name a inside the namespace .
+  I couldn't figure out what a refers to here:
   
       2 |   (a,b) = (a Nat.+ b, 19)
   
+  Its type should conform to Nat .
```

Comment:
Maybe either remove the `.` or drop the space :)

Reply:
This is `wrap` adding spaces again